tfsdk/config_test.go

```diff
@@ -329,6 +329,66 @@ func TestConfigGetAttribute(t *testing.T) {
 			path:     tftypes.NewAttributePath().WithAttributeName("test").WithElementKeyInt(0).WithAttributeName("sub_test"),
 			expected: types.String{Null: true},
 		},
+		"WithAttributeName-ListNestedAttributes-null-WithElementKeyInt-WithAttributeName-Object": {
```

Comment:
Looking at this now, I think the original problem was also possible if the nested item here was a list, map, or set instead of an object. Do you think it's worth testing those cases explicitly too or should the unit tests on the value types cover that?

Reply:
If you're inclined, the additional unit testing to ensure we testing in-depth would be appreciated, but I wouldn't want to hold up these fixes on it since its covered closer to the source.